.github/workflows/master.yml

```
@@ -10,7 +10,11 @@ jobs:
     name: Deploy
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
+    - name: auth
+      uses: google-github-actions/auth@v2
+      with:
+        credentials_json: ${{ secrets.GCP_SA_KEY }}
     - name: deploy
       uses: google-github-actions/deploy-cloud-functions@main
```

mainじゃなくてv2でmajor固定した方が今回のように突然壊れなくて良いはずです
```suggestion
      uses: google-github-actions/deploy-cloud-functions@v2
```